wrappers/ortools_result_pb.rb

```diff
@@ -15,7 +15,7 @@
     optional :lateness, :int64, 8
   end
 
-  add_message 'ortools_result.Costs' do
+  add_message 'ortools_result.CostDetails' do
```

Comment:
Doesn't this mean that we need to change the proto on the optimizer-ortools side as well?

Reply:
Costs are already renamed for a long time on Optimizer-ortools. As Costs and CostDetails have the same index and structure in the proto there it generates no conflict